azurerm/resource_arm_iothub.go

```diff
@@ -323,6 +323,32 @@ func resourceArmIotHub() *schema.Resource {
 				},
 			},
 
+			"ip_filter_rule": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
```

Comment:
Should we validate this with `NoEmptyStrings`?

Reply:
👍 agreed - since this otherwise LGTM I'm going to push a commit to add this so we can merge it, I hope you don't mind @Lucretius